src/components/SourceCodeEditor.tsx

```diff
@@ -57,24 +67,41 @@ const SourceCodeEditor = (props: ISourceCodeEditor) => {
   };
 
   const handleEditorDidMount = (editor: EditorDidMount['editor']) => {
+    const messageContribution: any = editor?.getContribution('editor.contrib.messageController');
```

Comment:
Quick question-- why `editor?` instead of just `editor`, considering it's a required parameter?

Reply:
thanks you're right